crates/cli/src/package.rs

```diff
@@ -14,10 +14,10 @@ use std::path::Path;
 /// symlink_path will be appended by the name of the package. Therefore,
 /// it should be resolved into the node_modules folder of a subdependency such as
 /// `node_modules/.pacquet/fastify@1.0.0/node_modules`.
-pub async fn find_package_version_from_registry(
+pub async fn install_package_from_registry(
```

Comment:
For future reference, I think we shouldn't put any unrelated changes to a pull request of this size.

Reply:
There are not related, **yet**